rules/internal/objc_provider_utils.bzl

```diff
@@ -55,29 +55,9 @@ def _merge_dynamic_framework_providers(dynamic_framework_providers):
 
     return apple_common.new_dynamic_framework_provider(**fields)
 
-def _merge_cc_info_providers(cc_info_providers, merge_keys = [
```

Comment:
Actually this is not used anywhere else.

Reply:
Overall this is "internal" but this overall utility module clearly has many useful functions none the less - we should clarify API contracts with this code.